sqlglot/dialects/singlestore.py

```diff
@@ -50,8 +50,26 @@ class SingleStore(Dialect):
         "DATABASE": "SCHEMA"
     }
 
-    # TODO: implement
-    # class Tokenizer(tokens.Tokenizer):
+    class Tokenizer(tokens.Tokenizer):
+        BIT_STRINGS = [("b'", "'"), ("B'", "'"), ("0b", "")]
+        HEX_STRINGS = [("x'", "'"), ("X'", "'"), ("0x", "")]
+        IDENTIFIERS = ['"', '`']
+        QUOTES = ["'", '"']
+        STRING_ESCAPES = ["'", '"', "\\"]
+        COMMENTS = ["--", "#", ("/*", "*/")]
+
+        KEYWORDS = {
+            **Tokenizer.KEYWORDS,
+            "@@": TokenType.SESSION_PARAMETER,
+            "YEAR": TokenType.YEAR,
+            "BSON": TokenType.JSONB,
+            "GEOGRAPHYPOINT": TokenType.GEOGRAPHY,
+            "IGNORE": TokenType.IGNORE,
+            "KEY": TokenType.KEY,
+            "START": TokenType.BEGIN
+        }
+
+        COMMANDS = {*tokens.Tokenizer.COMMANDS, TokenType.REPLACE} - {TokenType.SHOW}
```

Comment:
why do we remove `{TokenType.SHOW}` from the `COMMANDS`?

Reply:
I copied this behaviour from MySQL.
sqlglot parse all statements that start with COMMAND as a single node in AST and won't translate it to other dialects https://github.com/tobymao/sqlglot/blob/main/posts/onboarding.md#command-fallback .
As for the SHOW command, MySQL connector has logic to convert SHOW commands to general queries.
It makes sense to implement the same for SingleStore.